### ISSUE TITLE
Impress Notebookbar UI not compliant with the rest plus missing Edit Master Slide

### ISSUE THREAD

**Issue by pedropintosilva** [CollaboraOnline/online#4633]
**Describe the bug**

**To Reproduce**
1. Open any presentation document
2. Navigate to View Tab
3. Only Sidebar properties trigger is present, missing all the other sidebar views (in all the other apps sidebar view is present in this tab). On top of that users from LO also expect to see it in the View tab

---

1. Open any presentation document
2. Navigate to Layout Tab
3. View options are unexpected visible here (sidebar buttons)

---

1. Open any presentation document
2. Navigate to Layout Tab
3. Change to Master Slide View is missing (where you can edit Master slides). It's not visible anywhere else either. (we used to have it in the hamburger menu)

**Expected behavior**
* Remove ambiguous "Slide Layout" from Layout tab
* Move All the Sidebar buttons to View tab
* Add "Master Slide" view button to the View tab
* Extra bonus: change Toggle Status bar to bigtoolitem
![image](https://user-images.githubusercontent.com/65948705/164407786-d48c3484-30b0-4194-9eb4-cf25d90b9f87.png)

**Desktop (please complete the following information)**
- Master branch, desktop, using Notebookbar UI mode

**Comment by thebearon** [comment]
There's a significant difference between LO desktop and online: online has toggles for the different available sidebar views in the Notebookbar.
Ie. by moving the three buttons to `View`, and keeping the existing button as `Sidebar`, the naming for `Sidebar` would be ambiguous, as it only refers to the `Properties` view of the sidebar.

At first I thought it could be named `Slide Layout`, as the button-to-be-removed on the `Layout` tab, but perhaps `Properties` would be a better name? @pedropintosilva, what do you think?

**Comment by pedropintosilva** [comment]
**update, added to the main description**

excerpt taken from https://github.com/CollaboraOnline/online/pull/4782#pullrequestreview-981566183

I would prefer to have the view button to the the first and the sidebar the last because:
1. Then the Sidebar icon is the rightmost icon which is nice since the sidebar comes from that sode
2. The view button would be in the leftmost side from those icons. That's ideal since it's an action that is not only hierarchy above the other actions in that group of buttons but also probably more important to the user to spot straight away 

TLDR: 
![image](https://user-images.githubusercontent.com/65948705/169805439-b94bb31f-82e4-4fbf-9495-19f40fc09865.png)

**Comment by thebearon** [comment]
@pedropintosilva I'm a bit confused between this line from the initial description:
> Move All the Sidebar buttons to View tab

...and this comment here:
https://github.com/CollaboraOnline/online/pull/4782#discussion_r882558328

**Comment by pedropintosilva** [comment]
This has been already done
